package.json

```diff
@@ -21,7 +21,8 @@
 	"devDependencies": {
 		"@codeyourfuture/eslint-config-standard": "^4.0.3",
 		"concurrently": "^8.2.2",
-		"cross-env": "^7.0.3"
+		"cross-env": "^7.0.3",
+		"nodemon": "^3.1.0"
```

Comment:
We want to keep this project as simple as possible, hence nodemon was explicitly not included